// phosphor-vue/dist/esm/src/entry.esm.js
import * as index from './components/index.js';
export { default as PhActivity } from './components/PhActivity.vue.js';
export { default as PhAddressBook } from './components/PhAddressBook.vue.js';
export { default as PhAirplane } from './components/PhAirplane.vue.js';
export { default as PhAirplaneInFlight } from './components/PhAirplaneInFlight.vue.js';
export { default as PhAirplaneLanding } from './components/PhAirplaneLanding.vue.js';
export { default as PhAirplaneTakeoff } from './components/PhAirplaneTakeoff.vue.js';
export { default as PhAirplaneTilt } from './components/PhAirplaneTilt.vue.js';
export { default as PhAirplay } from './components/PhAirplay.vue.js';
export { default as PhAlarm } from './components/PhAlarm.vue.js';
export { default as PhAlien } from './components/PhAlien.vue.js';
export { default as PhAlignBottom } from './components/PhAlignBottom.vue.js';
export { default as PhAlignBottomSimple } from './components/PhAlignBottomSimple.vue.js';
export { default as PhAlignCenterHorizontal } from './components/PhAlignCenterHorizontal.vue.js';
export { default as PhAlignCenterHorizontalSimple } from './components/PhAlignCenterHorizontalSimple.vue.js';
export { default as PhAlignCenterVertical } from './components/PhAlignCenterVertical.vue.js';
export { default as PhAlignCenterVerticalSimple } from './components/PhAlignCenterVerticalSimple.vue.js';
export { default as PhAlignLeft } from './components/PhAlignLeft.vue.js';
export { default as PhAlignLeftSimple } from './components/PhAlignLeftSimple.vue.js';
export { default as PhAlignRight } from './components/PhAlignRight.vue.js';
export { default as PhAlignRightSimple } from './components/PhAlignRightSimple.vue.js';
export { default as PhAlignTop } from './components/PhAlignTop.vue.js';
export { default as PhAlignTopSimple } from './components/PhAlignTopSimple.vue.js';
export { default as PhAnchor } from './components/PhAnchor.vue.js';
export { default as PhAnchorSimple } from './components/PhAnchorSimple.vue.js';
export { default as PhAndroidLogo } from './components/PhAndroidLogo.vue.js';
export { default as PhAngularLogo } from './components/PhAngularLogo.vue.js';
export { default as PhAperture } from './components/PhAperture.vue.js';
export { default as PhAppStoreLogo } from './components/PhAppStoreLogo.vue.js';
export { default as PhAppWindow } from './components/PhAppWindow.vue.js';
export { default as PhAppleLogo } from './components/PhAppleLogo.vue.js';
export { default as PhApplePodcastsLogo } from './components/PhApplePodcastsLogo.vue.js';
export { default as PhArchive } from './components/PhArchive.vue.js';
export { default as PhArchiveBox } from './components/PhArchiveBox.vue.js';
export { default as PhArchiveTray } from './components/PhArchiveTray.vue.js';
export { default as PhArmchair } from './components/PhArmchair.vue.js';
export { default as PhArrowArcLeft } from './components/PhArrowArcLeft.vue.js';
export { default as PhArrowArcRight } from './components/PhArrowArcRight.vue.js';
export { default as PhArrowBendDoubleUpLeft } from './components/PhArrowBendDoubleUpLeft.vue.js';
export { default as PhArrowBendDoubleUpRight } from './components/PhArrowBendDoubleUpRight.vue.js';
export { default as PhArrowBendDownLeft } from './components/PhArrowBendDownLeft.vue.js';
export { default as PhArrowBendDownRight } from './components/PhArrowBendDownRight.vue.js';
export { default as PhArrowBendLeftDown } from './components/PhArrowBendLeftDown.vue.js';
export { default as PhArrowBendLeftUp } from './components/PhArrowBendLeftUp.vue.js';
export { default as PhArrowBendRightDown } from './components/PhArrowBendRightDown.vue.js';
export { default as PhArrowBendRightUp } from './components/PhArrowBendRightUp.vue.js';
export { default as PhArrowBendUpLeft } from './components/PhArrowBendUpLeft.vue.js';
export { default as PhArrowBendUpRight } from './components/PhArrowBendUpRight.vue.js';
export { default as PhArrowCircleDown } from './components/PhArrowCircleDown.vue.js';
export { default as PhArrowCircleDownLeft } from './components/PhArrowCircleDownLeft.vue.js';
export { default as PhArrowCircleDownRight } from './components/PhArrowCircleDownRight.vue.js';
export { default as PhArrowCircleLeft } from './components/PhArrowCircleLeft.vue.js';
export { default as PhArrowCircleRight } from './components/PhArrowCircleRight.vue.js';
export { default as PhArrowCircleUp } from './components/PhArrowCircleUp.vue.js';
export { default as PhArrowCircleUpLeft } from './components/PhArrowCircleUpLeft.vue.js';
export { default as PhArrowCircleUpRight } from './components/PhArrowCircleUpRight.vue.js';
export { default as PhArrowClockwise } from './components/PhArrowClockwise.vue.js';
export { default as PhArrowCounterClockwise } from './components/PhArrowCounterClockwise.vue.js';
export { default as PhArrowDown } from './components/PhArrowDown.vue.js';
export { default as PhArrowDownLeft } from './components/PhArrowDownLeft.vue.js';
export { default as PhArrowDownRight } from './components/PhArrowDownRight.vue.js';
export { default as PhArrowElbowDownLeft } from './components/PhArrowElbowDownLeft.vue.js';
export { default as PhArrowElbowDownRight } from './components/PhArrowElbowDownRight.vue.js';
export { default as PhArrowElbowLeft } from './components/PhArrowElbowLeft.vue.js';
export { default as PhArrowElbowLeftDown } from './components/PhArrowElbowLeftDown.vue.js';
export { default as PhArrowElbowLeftUp } from './components/PhArrowElbowLeftUp.vue.js';
export { default as PhArrowElbowRight } from './components/PhArrowElbowRight.vue.js';
export { default as PhArrowElbowRightDown } from './components/PhArrowElbowRightDown.vue.js';
export { default as PhArrowElbowRightUp } from './components/PhArrowElbowRightUp.vue.js';
export { default as PhArrowElbowUpLeft } from './components/PhArrowElbowUpLeft.vue.js';
export { default as PhArrowElbowUpRight } from './components/PhArrowElbowUpRight.vue.js';
export { default as PhArrowFatDown } from './components/PhArrowFatDown.vue.js';
export { default as PhArrowFatLeft } from './components/PhArrowFatLeft.vue.js';
export { default as PhArrowFatLineDown } from './components/PhArrowFatLineDown.vue.js';
export { default as PhArrowFatLineLeft } from './components/PhArrowFatLineLeft.vue.js';
export { default as PhArrowFatLineRight } from './components/PhArrowFatLineRight.vue.js';
export { default as PhArrowFatLineUp } from './components/PhArrowFatLineUp.vue.js';
export { default as PhArrowFatLinesDown } from './components/PhArrowFatLinesDown.vue.js';
export { default as PhArrowFatLinesLeft } from './components/PhArrowFatLinesLeft.vue.js';
export { default as PhArrowFatLinesRight } from './components/PhArrowFatLinesRight.vue.js';
export { default as PhArrowFatLinesUp } from './components/PhArrowFatLinesUp.vue.js';
export { default as PhArrowFatRight } from './components/PhArrowFatRight.vue.js';
export { default as PhArrowFatUp } from './components/PhArrowFatUp.vue.js';
export { default as PhArrowLeft } from './components/PhArrowLeft.vue.js';
export { default as PhArrowLineDown } from './components/PhArrowLineDown.vue.js';
export { default as PhArrowLineDownLeft } from './components/PhArrowLineDownLeft.vue.js';
export { default as PhArrowLineDownRight } from './components/PhArrowLineDownRight.vue.js';
export { default as PhArrowLineLeft } from './components/PhArrowLineLeft.vue.js';
export { default as PhArrowLineRight } from './components/PhArrowLineRight.vue.js';
export { default as PhArrowLineUp } from './components/PhArrowLineUp.vue.js';
export { default as PhArrowLineUpLeft } from './components/PhArrowLineUpLeft.vue.js';
export { default as PhArrowLineUpRight } from './components/PhArrowLineUpRight.vue.js';
export { default as PhArrowRight } from './components/PhArrowRight.vue.js';
export { default as PhArrowSquareDown } from './components/PhArrowSquareDown.vue.js';
export { default as PhArrowSquareDownLeft } from './components/PhArrowSquareDownLeft.vue.js';
export { default as PhArrowSquareDownRight } from './components/PhArrowSquareDownRight.vue.js';
export { default as PhArrowSquareIn } from './components/PhArrowSquareIn.vue.js';
export { default as PhArrowSquareLeft } from './components/PhArrowSquareLeft.vue.js';
export { default as PhArrowSquareOut } from './components/PhArrowSquareOut.vue.js';
export { default as PhArrowSquareRight } from './components/PhArrowSquareRight.vue.js';
export { default as PhArrowSquareUp } from './components/PhArrowSquareUp.vue.js';
export { default as PhArrowSquareUpLeft } from './components/PhArrowSquareUpLeft.vue.js';
export { default as PhArrowSquareUpRight } from './components/PhArrowSquareUpRight.vue.js';
export { default as PhArrowUDownLeft } from './components/PhArrowUDownLeft.vue.js';
export { default as PhArrowUDownRight } from './components/PhArrowUDownRight.vue.js';
export { default as PhArrowULeftDown } from './components/PhArrowULeftDown.vue.js';
export { default as PhArrowULeftUp } from './components/PhArrowULeftUp.vue.js';
export { default as PhArrowURightDown } from './components/PhArrowURightDown.vue.js';
export { default as PhArrowURightUp } from './components/PhArrowURightUp.vue.js';
export { default as PhArrowUUpLeft } from './components/PhArrowUUpLeft.vue.js';
export { default as PhArrowUUpRight } from './components/PhArrowUUpRight.vue.js';
export { default as PhArrowUp } from './components/PhArrowUp.vue.js';
export { default as PhArrowUpLeft } from './components/PhArrowUpLeft.vue.js';
export { default as PhArrowUpRight } from './components/PhArrowUpRight.vue.js';
export { default as PhArrowsClockwise } from './components/PhArrowsClockwise.vue.js';
export { default as PhArrowsCounterClockwise } from './components/PhArrowsCounterClockwise.vue.js';
export { default as PhArrowsDownUp } from './components/PhArrowsDownUp.vue.js';
export { default as PhArrowsHorizontal } from './components/PhArrowsHorizontal.vue.js';
export { default as PhArrowsIn } from './components/PhArrowsIn.vue.js';
export { default as PhArrowsInCardinal } from './components/PhArrowsInCardinal.vue.js';
export { default as PhArrowsInLineHorizontal } from './components/PhArrowsInLineHorizontal.vue.js';
export { default as PhArrowsInLineVertical } from './components/PhArrowsInLineVertical.vue.js';
export { default as PhArrowsInSimple } from './components/PhArrowsInSimple.vue.js';
export { default as PhArrowsLeftRight } from './components/PhArrowsLeftRight.vue.js';
export { default as PhArrowsOut } from './components/PhArrowsOut.vue.js';
export { default as PhArrowsOutCardinal } from './components/PhArrowsOutCardinal.vue.js';
export { default as PhArrowsOutLineHorizontal } from './components/PhArrowsOutLineHorizontal.vue.js';
export { default as PhArrowsOutLineVertical } from './components/PhArrowsOutLineVertical.vue.js';
export { default as PhArrowsOutSimple } from './components/PhArrowsOutSimple.vue.js';
export { default as PhArrowsVertical } from './components/PhArrowsVertical.vue.js';
export { default as PhArticle } from './components/PhArticle.vue.js';
export { default as PhArticleMedium } from './components/PhArticleMedium.vue.js';
export { default as PhArticleNyTimes } from './components/PhArticleNyTimes.vue.js';
export { default as PhAsterisk } from './components/PhAsterisk.vue.js';
export { default as PhAsteriskSimple } from './components/PhAsteriskSimple.vue.js';
export { default as PhAt } from './components/PhAt.vue.js';
export { default as PhAtom } from './components/PhAtom.vue.js';
export { default as PhBaby } from './components/PhBaby.vue.js';
export { default as PhBackpack } from './components/PhBackpack.vue.js';
export { default as PhBackspace } from './components/PhBackspace.vue.js';
export { default as PhBag } from './components/PhBag.vue.js';
export { default as PhBagSimple } from './components/PhBagSimple.vue.js';
export { default as PhBalloon } from './components/PhBalloon.vue.js';
export { default as PhBandaids } from './components/PhBandaids.vue.js';
export { default as PhBank } from './components/PhBank.vue.js';
export { default as PhBarbell } from './components/PhBarbell.vue.js';
export { default as PhBarcode } from './components/PhBarcode.vue.js';
export { default as PhBarricade } from './components/PhBarricade.vue.js';
export { default as PhBaseball } from './components/PhBaseball.vue.js';
export { default as PhBasketball } from './components/PhBasketball.vue.js';
export { default as PhBathtub } from './components/PhBathtub.vue.js';
export { default as PhBatteryCharging } from './components/PhBatteryCharging.vue.js';
export { default as PhBatteryChargingVertical } from './components/PhBatteryChargingVertical.vue.js';
export { default as PhBatteryEmpty } from './components/PhBatteryEmpty.vue.js';
export { default as PhBatteryFull } from './components/PhBatteryFull.vue.js';
export { default as PhBatteryHigh } from './components/PhBatteryHigh.vue.js';
export { default as PhBatteryLow } from './components/PhBatteryLow.vue.js';
export { default as PhBatteryMedium } from './components/PhBatteryMedium.vue.js';
export { default as PhBatteryPlus } from './components/PhBatteryPlus.vue.js';
export { default as PhBatteryWarning } from './components/PhBatteryWarning.vue.js';
export { default as PhBatteryWarningVertical } from './components/PhBatteryWarningVertical.vue.js';
export { default as PhBed } from './components/PhBed.vue.js';
export { default as PhBeerBottle } from './components/PhBeerBottle.vue.js';
export { default as PhBehanceLogo } from './components/PhBehanceLogo.vue.js';
export { default as PhBell } from './components/PhBell.vue.js';
export { default as PhBellRinging } from './components/PhBellRinging.vue.js';
export { default as PhBellSimple } from './components/PhBellSimple.vue.js';
export { default as PhBellSimpleRinging } from './components/PhBellSimpleRinging.vue.js';
export { default as PhBellSimpleSlash } from './components/PhBellSimpleSlash.vue.js';
export { default as PhBellSimpleZ } from './components/PhBellSimpleZ.vue.js';
export { default as PhBellSlash } from './components/PhBellSlash.vue.js';
export { default as PhBellZ } from './components/PhBellZ.vue.js';
export { default as PhBezierCurve } from './components/PhBezierCurve.vue.js';
export { default as PhBicycle } from './components/PhBicycle.vue.js';
export { default as PhBinoculars } from './components/PhBinoculars.vue.js';
export { default as PhBird } from './components/PhBird.vue.js';
export { default as PhBluetooth } from './components/PhBluetooth.vue.js';
export { default as PhBluetoothConnected } from './components/PhBluetoothConnected.vue.js';
export { default as PhBluetoothSlash } from './components/PhBluetoothSlash.vue.js';
export { default as PhBluetoothX } from './components/PhBluetoothX.vue.js';
export { default as PhBoat } from './components/PhBoat.vue.js';
export { default as PhBook } from './components/PhBook.vue.js';
export { default as PhBookBookmark } from './components/PhBookBookmark.vue.js';
export { default as PhBookOpen } from './components/PhBookOpen.vue.js';
export { default as PhBookmark } from './components/PhBookmark.vue.js';
export { default as PhBookmarkSimple } from './components/PhBookmarkSimple.vue.js';
export { default as PhBookmarks } from './components/PhBookmarks.vue.js';
export { default as PhBookmarksSimple } from './components/PhBookmarksSimple.vue.js';
export { default as PhBooks } from './components/PhBooks.vue.js';
export { default as PhBoundingBox } from './components/PhBoundingBox.vue.js';
export { default as PhBracketsAngle } from './components/PhBracketsAngle.vue.js';
export { default as PhBracketsCurly } from './components/PhBracketsCurly.vue.js';
export { default as PhBracketsRound } from './components/PhBracketsRound.vue.js';
export { default as PhBracketsSquare } from './components/PhBracketsSquare.vue.js';
export { default as PhBrain } from './components/PhBrain.vue.js';
export { default as PhBrandy } from './components/PhBrandy.vue.js';
export { default as PhBriefcase } from './components/PhBriefcase.vue.js';
export { default as PhBriefcaseMetal } from './components/PhBriefcaseMetal.vue.js';
export { default as PhBroadcast } from './components/PhBroadcast.vue.js';
export { default as PhBrowser } from './components/PhBrowser.vue.js';
export { default as PhBrowsers } from './components/PhBrowsers.vue.js';
export { default as PhBug } from './components/PhBug.vue.js';
export { default as PhBugBeetle } from './components/PhBugBeetle.vue.js';
export { default as PhBugDroid } from './components/PhBugDroid.vue.js';
export { default as PhBuildings } from './components/PhBuildings.vue.js';
export { default as PhBus } from './components/PhBus.vue.js';
export { default as PhButterfly } from './components/PhButterfly.vue.js';
export { default as PhCactus } from './components/PhCactus.vue.js';
export { default as PhCake } from './components/PhCake.vue.js';
export { default as PhCalculator } from './components/PhCalculator.vue.js';
export { default as PhCalendar } from './components/PhCalendar.vue.js';
export { default as PhCalendarBlank } from './components/PhCalendarBlank.vue.js';
export { default as PhCalendarCheck } from './components/PhCalendarCheck.vue.js';
export { default as PhCalendarPlus } from './components/PhCalendarPlus.vue.js';
export { default as PhCalendarX } from './components/PhCalendarX.vue.js';
export { default as PhCamera } from './components/PhCamera.vue.js';
export { default as PhCameraRotate } from './components/PhCameraRotate.vue.js';
export { default as PhCameraSlash } from './components/PhCameraSlash.vue.js';
export { default as PhCampfire } from './components/PhCampfire.vue.js';
export { default as PhCar } from './components/PhCar.vue.js';
export { default as PhCarSimple } from './components/PhCarSimple.vue.js';
export { default as PhCardholder } from './components/PhCardholder.vue.js';
export { default as PhCards } from './components/PhCards.vue.js';
export { default as PhCaretCircleDoubleDown } from './components/PhCaretCircleDoubleDown.vue.js';
export { default as PhCaretCircleDoubleLeft } from './components/PhCaretCircleDoubleLeft.vue.js';
export { default as PhCaretCircleDoubleRight } from './components/PhCaretCircleDoubleRight.vue.js';
export { default as PhCaretCircleDoubleUp } from './components/PhCaretCircleDoubleUp.vue.js';
export { default as PhCaretCircleDown } from './components/PhCaretCircleDown.vue.js';
export { default as PhCaretCircleLeft } from './components/PhCaretCircleLeft.vue.js';
export { default as PhCaretCircleRight } from './components/PhCaretCircleRight.vue.js';
export { default as PhCaretCircleUp } from './components/PhCaretCircleUp.vue.js';
export { default as PhCaretDoubleDown } from './components/PhCaretDoubleDown.vue.js';
export { default as PhCaretDoubleLeft } from './components/PhCaretDoubleLeft.vue.js';
export { default as PhCaretDoubleRight } from './components/PhCaretDoubleRight.vue.js';
export { default as PhCaretDoubleUp } from './components/PhCaretDoubleUp.vue.js';
export { default as PhCaretDown } from './components/PhCaretDown.vue.js';
export { default as PhCaretLeft } from './components/PhCaretLeft.vue.js';
export { default as PhCaretRight } from './components/PhCaretRight.vue.js';
export { default as PhCaretUp } from './components/PhCaretUp.vue.js';
export { default as PhCat } from './components/PhCat.vue.js';
export { default as PhCellSignalFull } from './components/PhCellSignalFull.vue.js';
export { default as PhCellSignalHigh } from './components/PhCellSignalHigh.vue.js';
export { default as PhCellSignalLow } from './components/PhCellSignalLow.vue.js';
export { default as PhCellSignalMedium } from './components/PhCellSignalMedium.vue.js';
export { default as PhCellSignalNone } from './components/PhCellSignalNone.vue.js';
export { default as PhCellSignalSlash } from './components/PhCellSignalSlash.vue.js';
export { default as PhCellSignalX } from './components/PhCellSignalX.vue.js';
export { default as PhChalkboard } from './components/PhChalkboard.vue.js';
export { default as PhChalkboardSimple } from './components/PhChalkboardSimple.vue.js';
export { default as PhChalkboardTeacher } from './components/PhChalkboardTeacher.vue.js';
export { default as PhChartBar } from './components/PhChartBar.vue.js';
export { default as PhChartBarHorizontal } from './components/PhChartBarHorizontal.vue.js';
export { default as PhChartLine } from './components/PhChartLine.vue.js';
export { default as PhChartLineUp } from './components/PhChartLineUp.vue.js';
export { default as PhChartPie } from './components/PhChartPie.vue.js';
export { default as PhChartPieSlice } from './components/PhChartPieSlice.vue.js';
export { default as PhChat } from './components/PhChat.vue.js';
export { default as PhChatCentered } from './components/PhChatCentered.vue.js';
export { default as PhChatCenteredDots } from './components/PhChatCenteredDots.vue.js';
export { default as PhChatCenteredText } from './components/PhChatCenteredText.vue.js';
export { default as PhChatCircle } from './components/PhChatCircle.vue.js';
export { default as PhChatCircleDots } from './components/PhChatCircleDots.vue.js';
export { default as PhChatCircleText } from './components/PhChatCircleText.vue.js';
export { default as PhChatDots } from './components/PhChatDots.vue.js';
export { default as PhChatTeardrop } from './components/PhChatTeardrop.vue.js';
export { default as PhChatTeardropDots } from './components/PhChatTeardropDots.vue.js';
export { default as PhChatTeardropText } from './components/PhChatTeardropText.vue.js';
export { default as PhChatText } from './components/PhChatText.vue.js';
export { default as PhChats } from './components/PhChats.vue.js';
export { default as PhChatsCircle } from './components/PhChatsCircle.vue.js';
export { default as PhChatsTeardrop } from './components/PhChatsTeardrop.vue.js';
export { default as PhCheck } from './components/PhCheck.vue.js';
export { default as PhCheckCircle } from './components/PhCheckCircle.vue.js';
export { default as PhCheckSquare } from './components/PhCheckSquare.vue.js';
export { default as PhCheckSquareOffset } from './components/PhCheckSquareOffset.vue.js';
export { default as PhChecks } from './components/PhChecks.vue.js';
export { default as PhCircle } from './components/PhCircle.vue.js';
export { default as PhCircleDashed } from './components/PhCircleDashed.vue.js';
export { default as PhCircleHalf } from './components/PhCircleHalf.vue.js';
export { default as PhCircleHalfTilt } from './components/PhCircleHalfTilt.vue.js';
export { default as PhCircleNotch } from './components/PhCircleNotch.vue.js';
export { default as PhCircleWavy } from './components/PhCircleWavy.vue.js';
export { default as PhCircleWavyCheck } from './components/PhCircleWavyCheck.vue.js';
export { default as PhCircleWavyQuestion } from './components/PhCircleWavyQuestion.vue.js';
export { default as PhCircleWavyWarning } from './components/PhCircleWavyWarning.vue.js';
export { default as PhCirclesFour } from './components/PhCirclesFour.vue.js';
export { default as PhCirclesThree } from './components/PhCirclesThree.vue.js';
export { default as PhCirclesThreePlus } from './components/PhCirclesThreePlus.vue.js';
export { default as PhClipboard } from './components/PhClipboard.vue.js';
export { default as PhClipboardText } from './components/PhClipboardText.vue.js';
export { default as PhClock } from './components/PhClock.vue.js';
export { default as PhClockAfternoon } from './components/PhClockAfternoon.vue.js';
export { default as PhClockClockwise } from './components/PhClockClockwise.vue.js';
export { default as PhClockCounterClockwise } from './components/PhClockCounterClockwise.vue.js';
export { default as PhClosedCaptioning } from './components/PhClosedCaptioning.vue.js';
export { default as PhCloud } from './components/PhCloud.vue.js';
export { default as PhCloudArrowDown } from './components/PhCloudArrowDown.vue.js';
export { default as PhCloudArrowUp } from './components/PhCloudArrowUp.vue.js';
export { default as PhCloudCheck } from './components/PhCloudCheck.vue.js';
export { default as PhCloudFog } from './components/PhCloudFog.vue.js';
export { default as PhCloudLightning } from './components/PhCloudLightning.vue.js';
export { default as PhCloudMoon } from './components/PhCloudMoon.vue.js';
export { default as PhCloudRain } from './components/PhCloudRain.vue.js';
export { default as PhCloudSlash } from './components/PhCloudSlash.vue.js';
export { default as PhCloudSnow } from './components/PhCloudSnow.vue.js';
export { default as PhCloudSun } from './components/PhCloudSun.vue.js';
export { default as PhClub } from './components/PhClub.vue.js';
export { default as PhCoatHanger } from './components/PhCoatHanger.vue.js';
export { default as PhCode } from './components/PhCode.vue.js';
export { default as PhCodeSimple } from './components/PhCodeSimple.vue.js';
export { default as PhCodepenLogo } from './components/PhCodepenLogo.vue.js';
export { default as PhCodesandboxLogo } from './components/PhCodesandboxLogo.vue.js';
export { default as PhCoffee } from './components/PhCoffee.vue.js';
export { default as PhCoin } from './components/PhCoin.vue.js';
export { default as PhCoinVertical } from './components/PhCoinVertical.vue.js';
export { default as PhCoins } from './components/PhCoins.vue.js';
export { default as PhColumns } from './components/PhColumns.vue.js';
export { default as PhCommand } from './components/PhCommand.vue.js';
export { default as PhCompass } from './components/PhCompass.vue.js';
export { default as PhComputerTower } from './components/PhComputerTower.vue.js';
export { default as PhConfetti } from './components/PhConfetti.vue.js';
export { default as PhCookie } from './components/PhCookie.vue.js';
export { default as PhCookingPot } from './components/PhCookingPot.vue.js';
export { default as PhCopy } from './components/PhCopy.vue.js';
export { default as PhCopySimple } from './components/PhCopySimple.vue.js';
export { default as PhCopyleft } from './components/PhCopyleft.vue.js';
export { default as PhCopyright } from './components/PhCopyright.vue.js';
export { default as PhCornersIn } from './components/PhCornersIn.vue.js';
export { default as PhCornersOut } from './components/PhCornersOut.vue.js';
export { default as PhCpu } from './components/PhCpu.vue.js';
export { default as PhCreditCard } from './components/PhCreditCard.vue.js';
export { default as PhCrop } from './components/PhCrop.vue.js';
export { default as PhCrosshair } from './components/PhCrosshair.vue.js';
export { default as PhCrosshairSimple } from './components/PhCrosshairSimple.vue.js';
export { default as PhCrown } from './components/PhCrown.vue.js';
export { default as PhCrownSimple } from './components/PhCrownSimple.vue.js';
export { default as PhCube } from './components/PhCube.vue.js';
export { default as PhCurrencyBtc } from './components/PhCurrencyBtc.vue.js';
export { default as PhCurrencyCircleDollar } from './components/PhCurrencyCircleDollar.vue.js';
export { default as PhCurrencyCny } from './components/PhCurrencyCny.vue.js';
export { default as PhCurrencyDollar } from './components/PhCurrencyDollar.vue.js';
export { default as PhCurrencyDollarSimple } from './components/PhCurrencyDollarSimple.vue.js';
export { default as PhCurrencyEth } from './components/PhCurrencyEth.vue.js';
export { default as PhCurrencyEur } from './components/PhCurrencyEur.vue.js';
export { default as PhCurrencyGbp } from './components/PhCurrencyGbp.vue.js';
export { default as PhCurrencyInr } from './components/PhCurrencyInr.vue.js';
export { default as PhCurrencyJpy } from './components/PhCurrencyJpy.vue.js';
export { default as PhCurrencyKrw } from './components/PhCurrencyKrw.vue.js';
export { default as PhCurrencyKzt } from './components/PhCurrencyKzt.vue.js';
export { default as PhCurrencyNgn } from './components/PhCurrencyNgn.vue.js';
export { default as PhCurrencyRub } from './components/PhCurrencyRub.vue.js';
export { default as PhCursor } from './components/PhCursor.vue.js';
export { default as PhCursorText } from './components/PhCursorText.vue.js';
export { default as PhCylinder } from './components/PhCylinder.vue.js';
export { default as PhDatabase } from './components/PhDatabase.vue.js';
export { default as PhDesktop } from './components/PhDesktop.vue.js';
export { default as PhDesktopTower } from './components/PhDesktopTower.vue.js';
export { default as PhDetective } from './components/PhDetective.vue.js';
export { default as PhDeviceMobile } from './components/PhDeviceMobile.vue.js';
export { default as PhDeviceMobileCamera } from './components/PhDeviceMobileCamera.vue.js';
export { default as PhDeviceMobileSpeaker } from './components/PhDeviceMobileSpeaker.vue.js';
export { default as PhDeviceTablet } from './components/PhDeviceTablet.vue.js';
export { default as PhDeviceTabletCamera } from './components/PhDeviceTabletCamera.vue.js';
export { default as PhDeviceTabletSpeaker } from './components/PhDeviceTabletSpeaker.vue.js';
export { default as PhDiamond } from './components/PhDiamond.vue.js';
export { default as PhDiamondsFour } from './components/PhDiamondsFour.vue.js';
export { default as PhDiceFive } from './components/PhDiceFive.vue.js';
export { default as PhDiceFour } from './components/PhDiceFour.vue.js';
export { default as PhDiceOne } from './components/PhDiceOne.vue.js';
export { default as PhDiceSix } from './components/PhDiceSix.vue.js';
export { default as PhDiceThree } from './components/PhDiceThree.vue.js';
export { default as PhDiceTwo } from './components/PhDiceTwo.vue.js';
export { default as PhDisc } from './components/PhDisc.vue.js';
export { default as PhDiscordLogo } from './components/PhDiscordLogo.vue.js';
export { default as PhDivide } from './components/PhDivide.vue.js';
export { default as PhDog } from './components/PhDog.vue.js';
export { default as PhDoor } from './components/PhDoor.vue.js';
export { default as PhDotsNine } from './components/PhDotsNine.vue.js';
export { default as PhDotsSix } from './components/PhDotsSix.vue.js';
export { default as PhDotsSixVertical } from './components/PhDotsSixVertical.vue.js';
export { default as PhDotsThree } from './components/PhDotsThree.vue.js';
export { default as PhDotsThreeCircle } from './components/PhDotsThreeCircle.vue.js';
export { default as PhDotsThreeCircleVertical } from './components/PhDotsThreeCircleVertical.vue.js';
export { default as PhDotsThreeOutline } from './components/PhDotsThreeOutline.vue.js';
export { default as PhDotsThreeOutlineVertical } from './components/PhDotsThreeOutlineVertical.vue.js';
export { default as PhDotsThreeVertical } from './components/PhDotsThreeVertical.vue.js';
export { default as PhDownload } from './components/PhDownload.vue.js';
export { default as PhDownloadSimple } from './components/PhDownloadSimple.vue.js';
export { default as PhDribbbleLogo } from './components/PhDribbbleLogo.vue.js';
export { default as PhDrop } from './components/PhDrop.vue.js';
export { default as PhDropHalf } from './components/PhDropHalf.vue.js';
export { default as PhDropHalfBottom } from './components/PhDropHalfBottom.vue.js';
export { default as PhEar } from './components/PhEar.vue.js';
export { default as PhEarSlash } from './components/PhEarSlash.vue.js';
export { default as PhEgg } from './components/PhEgg.vue.js';
export { default as PhEggCrack } from './components/PhEggCrack.vue.js';
export { default as PhEject } from './components/PhEject.vue.js';
export { default as PhEjectSimple } from './components/PhEjectSimple.vue.js';
export { default as PhEnvelope } from './components/PhEnvelope.vue.js';
export { default as PhEnvelopeOpen } from './components/PhEnvelopeOpen.vue.js';
export { default as PhEnvelopeSimple } from './components/PhEnvelopeSimple.vue.js';
export { default as PhEnvelopeSimpleOpen } from './components/PhEnvelopeSimpleOpen.vue.js';
export { default as PhEqualizer } from './components/PhEqualizer.vue.js';
export { default as PhEquals } from './components/PhEquals.vue.js';
export { default as PhEraser } from './components/PhEraser.vue.js';
export { default as PhExam } from './components/PhExam.vue.js';
export { default as PhExport } from './components/PhExport.vue.js';
export { default as PhEye } from './components/PhEye.vue.js';
export { default as PhEyeClosed } from './components/PhEyeClosed.vue.js';
export { default as PhEyeSlash } from './components/PhEyeSlash.vue.js';
export { default as PhEyedropper } from './components/PhEyedropper.vue.js';
export { default as PhEyedropperSample } from './components/PhEyedropperSample.vue.js';
export { default as PhEyeglasses } from './components/PhEyeglasses.vue.js';
export { default as PhFaceMask } from './components/PhFaceMask.vue.js';
export { default as PhFacebookLogo } from './components/PhFacebookLogo.vue.js';
export { default as PhFactory } from './components/PhFactory.vue.js';
export { default as PhFaders } from './components/PhFaders.vue.js';
export { default as PhFadersHorizontal } from './components/PhFadersHorizontal.vue.js';
export { default as PhFastForward } from './components/PhFastForward.vue.js';
export { default as PhFastForwardCircle } from './components/PhFastForwardCircle.vue.js';
export { default as PhFigmaLogo } from './components/PhFigmaLogo.vue.js';
export { default as PhFile } from './components/PhFile.vue.js';
export { default as PhFileArrowDown } from './components/PhFileArrowDown.vue.js';
export { default as PhFileArrowUp } from './components/PhFileArrowUp.vue.js';
export { default as PhFileAudio } from './components/PhFileAudio.vue.js';
export { default as PhFileCloud } from './components/PhFileCloud.vue.js';
export { default as PhFileCode } from './components/PhFileCode.vue.js';
export { default as PhFileCss } from './components/PhFileCss.vue.js';
export { default as PhFileCsv } from './components/PhFileCsv.vue.js';
export { default as PhFileDoc } from './components/PhFileDoc.vue.js';
export { default as PhFileDotted } from './components/PhFileDotted.vue.js';
export { default as PhFileHtml } from './components/PhFileHtml.vue.js';
export { default as PhFileImage } from './components/PhFileImage.vue.js';
export { default as PhFileJpg } from './components/PhFileJpg.vue.js';
export { default as PhFileJs } from './components/PhFileJs.vue.js';
export { default as PhFileJsx } from './components/PhFileJsx.vue.js';
export { default as PhFileLock } from './components/PhFileLock.vue.js';
export { default as PhFileMinus } from './components/PhFileMinus.vue.js';
export { default as PhFilePdf } from './components/PhFilePdf.vue.js';
export { default as PhFilePlus } from './components/PhFilePlus.vue.js';
export { default as PhFilePng } from './components/PhFilePng.vue.js';
export { default as PhFilePpt } from './components/PhFilePpt.vue.js';
export { default as PhFileRs } from './components/PhFileRs.vue.js';
export { default as PhFileSearch } from './components/PhFileSearch.vue.js';
export { default as PhFileText } from './components/PhFileText.vue.js';
export { default as PhFileTs } from './components/PhFileTs.vue.js';
export { default as PhFileTsx } from './components/PhFileTsx.vue.js';
export { default as PhFileVideo } from './components/PhFileVideo.vue.js';
export { default as PhFileVue } from './components/PhFileVue.vue.js';
export { default as PhFileX } from './components/PhFileX.vue.js';
export { default as PhFileXls } from './components/PhFileXls.vue.js';
export { default as PhFileZip } from './components/PhFileZip.vue.js';
export { default as PhFiles } from './components/PhFiles.vue.js';
export { default as PhFilmScript } from './components/PhFilmScript.vue.js';
export { default as PhFilmSlate } from './components/PhFilmSlate.vue.js';
export { default as PhFilmStrip } from './components/PhFilmStrip.vue.js';
export { default as PhFingerprint } from './components/PhFingerprint.vue.js';
export { default as PhFingerprintSimple } from './components/PhFingerprintSimple.vue.js';
export { default as PhFinnTheHuman } from './components/PhFinnTheHuman.vue.js';
export { default as PhFire } from './components/PhFire.vue.js';
export { default as PhFireSimple } from './components/PhFireSimple.vue.js';
export { default as PhFirstAid } from './components/PhFirstAid.vue.js';
export { default as PhFirstAidKit } from './components/PhFirstAidKit.vue.js';
export { default as PhFish } from './components/PhFish.vue.js';
export { default as PhFishSimple } from './components/PhFishSimple.vue.js';
export { default as PhFlag } from './components/PhFlag.vue.js';
export { default as PhFlagBanner } from './components/PhFlagBanner.vue.js';
export { default as PhFlagCheckered } from './components/PhFlagCheckered.vue.js';
export { default as PhFlame } from './components/PhFlame.vue.js';
export { default as PhFlashlight } from './components/PhFlashlight.vue.js';
export { default as PhFlask } from './components/PhFlask.vue.js';
export { default as PhFloppyDisk } from './components/PhFloppyDisk.vue.js';
export { default as PhFloppyDiskBack } from './components/PhFloppyDiskBack.vue.js';
export { default as PhFlowArrow } from './components/PhFlowArrow.vue.js';
export { default as PhFlower } from './components/PhFlower.vue.js';
export { default as PhFlowerLotus } from './components/PhFlowerLotus.vue.js';
export { default as PhFlyingSaucer } from './components/PhFlyingSaucer.vue.js';
export { default as PhFolder } from './components/PhFolder.vue.js';
export { default as PhFolderDotted } from './components/PhFolderDotted.vue.js';
export { default as PhFolderLock } from './components/PhFolderLock.vue.js';
export { default as PhFolderMinus } from './components/PhFolderMinus.vue.js';
export { default as PhFolderNotch } from './components/PhFolderNotch.vue.js';
export { default as PhFolderNotchMinus } from './components/PhFolderNotchMinus.vue.js';
export { default as PhFolderNotchOpen } from './components/PhFolderNotchOpen.vue.js';
export { default as PhFolderNotchPlus } from './components/PhFolderNotchPlus.vue.js';
export { default as PhFolderOpen } from './components/PhFolderOpen.vue.js';
export { default as PhFolderPlus } from './components/PhFolderPlus.vue.js';
export { default as PhFolderSimple } from './components/PhFolderSimple.vue.js';
export { default as PhFolderSimpleDotted } from './components/PhFolderSimpleDotted.vue.js';
export { default as PhFolderSimpleLock } from './components/PhFolderSimpleLock.vue.js';
export { default as PhFolderSimpleMinus } from './components/PhFolderSimpleMinus.vue.js';
export { default as PhFolderSimplePlus } from './components/PhFolderSimplePlus.vue.js';
export { default as PhFolderSimpleStar } from './components/PhFolderSimpleStar.vue.js';
export { default as PhFolderSimpleUser } from './components/PhFolderSimpleUser.vue.js';
export { default as PhFolderStar } from './components/PhFolderStar.vue.js';
export { default as PhFolderUser } from './components/PhFolderUser.vue.js';
export { default as PhFolders } from './components/PhFolders.vue.js';
export { default as PhFootball } from './components/PhFootball.vue.js';
export { default as PhForkKnife } from './components/PhForkKnife.vue.js';
export { default as PhFrameCorners } from './components/PhFrameCorners.vue.js';
export { default as PhFramerLogo } from './components/PhFramerLogo.vue.js';
export { default as PhFunction } from './components/PhFunction.vue.js';
export { default as PhFunnel } from './components/PhFunnel.vue.js';
export { default as PhFunnelSimple } from './components/PhFunnelSimple.vue.js';
export { default as PhGameController } from './components/PhGameController.vue.js';
export { default as PhGasPump } from './components/PhGasPump.vue.js';
export { default as PhGauge } from './components/PhGauge.vue.js';
export { default as PhGear } from './components/PhGear.vue.js';
export { default as PhGearSix } from './components/PhGearSix.vue.js';
export { default as PhGenderFemale } from './components/PhGenderFemale.vue.js';
export { default as PhGenderIntersex } from './components/PhGenderIntersex.vue.js';
export { default as PhGenderMale } from './components/PhGenderMale.vue.js';
export { default as PhGenderNeuter } from './components/PhGenderNeuter.vue.js';
export { default as PhGenderNonbinary } from './components/PhGenderNonbinary.vue.js';
export { default as PhGenderTransgender } from './components/PhGenderTransgender.vue.js';
export { default as PhGhost } from './components/PhGhost.vue.js';
export { default as PhGif } from './components/PhGif.vue.js';
export { default as PhGift } from './components/PhGift.vue.js';
export { default as PhGitBranch } from './components/PhGitBranch.vue.js';
export { default as PhGitCommit } from './components/PhGitCommit.vue.js';
export { default as PhGitDiff } from './components/PhGitDiff.vue.js';
export { default as PhGitFork } from './components/PhGitFork.vue.js';
export { default as PhGitMerge } from './components/PhGitMerge.vue.js';
export { default as PhGitPullRequest } from './components/PhGitPullRequest.vue.js';
export { default as PhGithubLogo } from './components/PhGithubLogo.vue.js';
export { default as PhGitlabLogo } from './components/PhGitlabLogo.vue.js';
export { default as PhGitlabLogoSimple } from './components/PhGitlabLogoSimple.vue.js';
export { default as PhGlobe } from './components/PhGlobe.vue.js';
export { default as PhGlobeHemisphereEast } from './components/PhGlobeHemisphereEast.vue.js';
export { default as PhGlobeHemisphereWest } from './components/PhGlobeHemisphereWest.vue.js';
export { default as PhGlobeSimple } from './components/PhGlobeSimple.vue.js';
export { default as PhGlobeStand } from './components/PhGlobeStand.vue.js';
export { default as PhGoogleChromeLogo } from './components/PhGoogleChromeLogo.vue.js';
export { default as PhGoogleLogo } from './components/PhGoogleLogo.vue.js';
export { default as PhGooglePhotosLogo } from './components/PhGooglePhotosLogo.vue.js';
export { default as PhGooglePlayLogo } from './components/PhGooglePlayLogo.vue.js';
export { default as PhGooglePodcastsLogo } from './components/PhGooglePodcastsLogo.vue.js';
export { default as PhGradient } from './components/PhGradient.vue.js';
export { default as PhGraduationCap } from './components/PhGraduationCap.vue.js';
export { default as PhGraph } from './components/PhGraph.vue.js';
export { default as PhGridFour } from './components/PhGridFour.vue.js';
export { default as PhHamburger } from './components/PhHamburger.vue.js';
export { default as PhHand } from './components/PhHand.vue.js';
export { default as PhHandEye } from './components/PhHandEye.vue.js';
export { default as PhHandFist } from './components/PhHandFist.vue.js';
export { default as PhHandGrabbing } from './components/PhHandGrabbing.vue.js';
export { default as PhHandPalm } from './components/PhHandPalm.vue.js';
export { default as PhHandPointing } from './components/PhHandPointing.vue.js';
export { default as PhHandSoap } from './components/PhHandSoap.vue.js';
export { default as PhHandWaving } from './components/PhHandWaving.vue.js';
export { default as PhHandbag } from './components/PhHandbag.vue.js';
export { default as PhHandbagSimple } from './components/PhHandbagSimple.vue.js';
export { default as PhHandsClapping } from './components/PhHandsClapping.vue.js';
export { default as PhHandshake } from './components/PhHandshake.vue.js';
export { default as PhHardDrive } from './components/PhHardDrive.vue.js';
export { default as PhHardDrives } from './components/PhHardDrives.vue.js';
export { default as PhHash } from './components/PhHash.vue.js';
export { default as PhHashStraight } from './components/PhHashStraight.vue.js';
export { default as PhHeadlights } from './components/PhHeadlights.vue.js';
export { default as PhHeadphones } from './components/PhHeadphones.vue.js';
export { default as PhHeadset } from './components/PhHeadset.vue.js';
export { default as PhHeart } from './components/PhHeart.vue.js';
export { default as PhHeartBreak } from './components/PhHeartBreak.vue.js';
export { default as PhHeartStraight } from './components/PhHeartStraight.vue.js';
export { default as PhHeartStraightBreak } from './components/PhHeartStraightBreak.vue.js';
export { default as PhHeartbeat } from './components/PhHeartbeat.vue.js';
export { default as PhHexagon } from './components/PhHexagon.vue.js';
export { default as PhHighlighterCircle } from './components/PhHighlighterCircle.vue.js';
export { default as PhHorse } from './components/PhHorse.vue.js';
export { default as PhHourglass } from './components/PhHourglass.vue.js';
export { default as PhHourglassHigh } from './components/PhHourglassHigh.vue.js';
export { default as PhHourglassLow } from './components/PhHourglassLow.vue.js';
export { default as PhHourglassMedium } from './components/PhHourglassMedium.vue.js';
export { default as PhHourglassSimple } from './components/PhHourglassSimple.vue.js';
export { default as PhHourglassSimpleHigh } from './components/PhHourglassSimpleHigh.vue.js';
export { default as PhHourglassSimpleLow } from './components/PhHourglassSimpleLow.vue.js';
export { default as PhHourglassSimpleMedium } from './components/PhHourglassSimpleMedium.vue.js';
export { default as PhHouse } from './components/PhHouse.vue.js';
export { default as PhHouseLine } from './components/PhHouseLine.vue.js';
export { default as PhHouseSimple } from './components/PhHouseSimple.vue.js';
export { default as PhIdentificationBadge } from './components/PhIdentificationBadge.vue.js';
export { default as PhIdentificationCard } from './components/PhIdentificationCard.vue.js';
export { default as PhImage } from './components/PhImage.vue.js';
export { default as PhImageSquare } from './components/PhImageSquare.vue.js';
export { default as PhInfinity } from './components/PhInfinity.vue.js';
export { default as PhInfo } from './components/PhInfo.vue.js';
export { default as PhInstagramLogo } from './components/PhInstagramLogo.vue.js';
export { default as PhIntersect } from './components/PhIntersect.vue.js';
export { default as PhJeep } from './components/PhJeep.vue.js';
export { default as PhKanban } from './components/PhKanban.vue.js';
export { default as PhKey } from './components/PhKey.vue.js';
export { default as PhKeyReturn } from './components/PhKeyReturn.vue.js';
export { default as PhKeyboard } from './components/PhKeyboard.vue.js';
export { default as PhKeyhole } from './components/PhKeyhole.vue.js';
export { default as PhKnife } from './components/PhKnife.vue.js';
export { default as PhLadder } from './components/PhLadder.vue.js';
export { default as PhLadderSimple } from './components/PhLadderSimple.vue.js';
export { default as PhLamp } from './components/PhLamp.vue.js';
export { default as PhLaptop } from './components/PhLaptop.vue.js';
export { default as PhLayout } from './components/PhLayout.vue.js';
export { default as PhLeaf } from './components/PhLeaf.vue.js';
export { default as PhLifebuoy } from './components/PhLifebuoy.vue.js';
export { default as PhLightbulb } from './components/PhLightbulb.vue.js';
export { default as PhLightbulbFilament } from './components/PhLightbulbFilament.vue.js';
export { default as PhLightning } from './components/PhLightning.vue.js';
export { default as PhLightningSlash } from './components/PhLightningSlash.vue.js';
export { default as PhLineSegment } from './components/PhLineSegment.vue.js';
export { default as PhLineSegments } from './components/PhLineSegments.vue.js';
export { default as PhLink } from './components/PhLink.vue.js';
export { default as PhLinkBreak } from './components/PhLinkBreak.vue.js';
export { default as PhLinkSimple } from './components/PhLinkSimple.vue.js';
export { default as PhLinkSimpleBreak } from './components/PhLinkSimpleBreak.vue.js';
export { default as PhLinkSimpleHorizontal } from './components/PhLinkSimpleHorizontal.vue.js';
export { default as PhLinkSimpleHorizontalBreak } from './components/PhLinkSimpleHorizontalBreak.vue.js';
export { default as PhLinkedinLogo } from './components/PhLinkedinLogo.vue.js';
export { default as PhLinuxLogo } from './components/PhLinuxLogo.vue.js';
export { default as PhList } from './components/PhList.vue.js';
export { default as PhListBullets } from './components/PhListBullets.vue.js';
export { default as PhListChecks } from './components/PhListChecks.vue.js';
export { default as PhListDashes } from './components/PhListDashes.vue.js';
export { default as PhListNumbers } from './components/PhListNumbers.vue.js';
export { default as PhListPlus } from './components/PhListPlus.vue.js';
export { default as PhLock } from './components/PhLock.vue.js';
export { default as PhLockKey } from './components/PhLockKey.vue.js';
export { default as PhLockKeyOpen } from './components/PhLockKeyOpen.vue.js';
export { default as PhLockLaminated } from './components/PhLockLaminated.vue.js';
export { default as PhLockLaminatedOpen } from './components/PhLockLaminatedOpen.vue.js';
export { default as PhLockOpen } from './components/PhLockOpen.vue.js';
export { default as PhLockSimple } from './components/PhLockSimple.vue.js';
export { default as PhLockSimpleOpen } from './components/PhLockSimpleOpen.vue.js';
export { default as PhMagicWand } from './components/PhMagicWand.vue.js';
export { default as PhMagnet } from './components/PhMagnet.vue.js';
export { default as PhMagnetStraight } from './components/PhMagnetStraight.vue.js';
export { default as PhMagnifyingGlass } from './components/PhMagnifyingGlass.vue.js';
export { default as PhMagnifyingGlassMinus } from './components/PhMagnifyingGlassMinus.vue.js';
export { default as PhMagnifyingGlassPlus } from './components/PhMagnifyingGlassPlus.vue.js';
export { default as PhMapPin } from './components/PhMapPin.vue.js';
export { default as PhMapPinLine } from './components/PhMapPinLine.vue.js';
export { default as PhMapTrifold } from './components/PhMapTrifold.vue.js';
export { default as PhMarkerCircle } from './components/PhMarkerCircle.vue.js';
export { default as PhMartini } from './components/PhMartini.vue.js';
export { default as PhMaskHappy } from './components/PhMaskHappy.vue.js';
export { default as PhMaskSad } from './components/PhMaskSad.vue.js';
export { default as PhMathOperations } from './components/PhMathOperations.vue.js';
export { default as PhMedal } from './components/PhMedal.vue.js';
export { default as PhMediumLogo } from './components/PhMediumLogo.vue.js';
export { default as PhMegaphone } from './components/PhMegaphone.vue.js';
export { default as PhMegaphoneSimple } from './components/PhMegaphoneSimple.vue.js';
export { default as PhMessengerLogo } from './components/PhMessengerLogo.vue.js';
export { default as PhMicrophone } from './components/PhMicrophone.vue.js';
export { default as PhMicrophoneSlash } from './components/PhMicrophoneSlash.vue.js';
export { default as PhMicrophoneStage } from './components/PhMicrophoneStage.vue.js';
export { default as PhMicrosoftExcelLogo } from './components/PhMicrosoftExcelLogo.vue.js';
export { default as PhMicrosoftPowerpointLogo } from './components/PhMicrosoftPowerpointLogo.vue.js';
export { default as PhMicrosoftTeamsLogo } from './components/PhMicrosoftTeamsLogo.vue.js';
export { default as PhMicrosoftWordLogo } from './components/PhMicrosoftWordLogo.vue.js';
export { default as PhMinus } from './components/PhMinus.vue.js';
export { default as PhMinusCircle } from './components/PhMinusCircle.vue.js';
export { default as PhMoney } from './components/PhMoney.vue.js';
export { default as PhMonitor } from './components/PhMonitor.vue.js';
export { default as PhMonitorPlay } from './components/PhMonitorPlay.vue.js';
export { default as PhMoon } from './components/PhMoon.vue.js';
export { default as PhMoonStars } from './components/PhMoonStars.vue.js';
export { default as PhMountains } from './components/PhMountains.vue.js';
export { default as PhMouse } from './components/PhMouse.vue.js';
export { default as PhMouseSimple } from './components/PhMouseSimple.vue.js';
export { default as PhMusicNote } from './components/PhMusicNote.vue.js';
export { default as PhMusicNoteSimple } from './components/PhMusicNoteSimple.vue.js';
export { default as PhMusicNotes } from './components/PhMusicNotes.vue.js';
export { default as PhMusicNotesPlus } from './components/PhMusicNotesPlus.vue.js';
export { default as PhMusicNotesSimple } from './components/PhMusicNotesSimple.vue.js';
export { default as PhNavigationArrow } from './components/PhNavigationArrow.vue.js';
export { default as PhNeedle } from './components/PhNeedle.vue.js';
export { default as PhNewspaper } from './components/PhNewspaper.vue.js';
export { default as PhNewspaperClipping } from './components/PhNewspaperClipping.vue.js';
export { default as PhNote } from './components/PhNote.vue.js';
export { default as PhNoteBlank } from './components/PhNoteBlank.vue.js';
export { default as PhNotePencil } from './components/PhNotePencil.vue.js';
export { default as PhNotebook } from './components/PhNotebook.vue.js';
export { default as PhNotepad } from './components/PhNotepad.vue.js';
export { default as PhNotification } from './components/PhNotification.vue.js';
export { default as PhNumberCircleEight } from './components/PhNumberCircleEight.vue.js';
export { default as PhNumberCircleFive } from './components/PhNumberCircleFive.vue.js';
export { default as PhNumberCircleFour } from './components/PhNumberCircleFour.vue.js';
export { default as PhNumberCircleNine } from './components/PhNumberCircleNine.vue.js';
export { default as PhNumberCircleOne } from './components/PhNumberCircleOne.vue.js';
export { default as PhNumberCircleSeven } from './components/PhNumberCircleSeven.vue.js';
export { default as PhNumberCircleSix } from './components/PhNumberCircleSix.vue.js';
export { default as PhNumberCircleThree } from './components/PhNumberCircleThree.vue.js';
export { default as PhNumberCircleTwo } from './components/PhNumberCircleTwo.vue.js';
export { default as PhNumberCircleZero } from './components/PhNumberCircleZero.vue.js';
export { default as PhNumberEight } from './components/PhNumberEight.vue.js';
export { default as PhNumberFive } from './components/PhNumberFive.vue.js';
export { default as PhNumberFour } from './components/PhNumberFour.vue.js';
export { default as PhNumberNine } from './components/PhNumberNine.vue.js';
export { default as PhNumberOne } from './components/PhNumberOne.vue.js';
export { default as PhNumberSeven } from './components/PhNumberSeven.vue.js';
export { default as PhNumberSix } from './components/PhNumberSix.vue.js';
export { default as PhNumberSquareEight } from './components/PhNumberSquareEight.vue.js';
export { default as PhNumberSquareFive } from './components/PhNumberSquareFive.vue.js';
export { default as PhNumberSquareFour } from './components/PhNumberSquareFour.vue.js';
export { default as PhNumberSquareNine } from './components/PhNumberSquareNine.vue.js';
export { default as PhNumberSquareOne } from './components/PhNumberSquareOne.vue.js';
export { default as PhNumberSquareSeven } from './components/PhNumberSquareSeven.vue.js';
export { default as PhNumberSquareSix } from './components/PhNumberSquareSix.vue.js';
export { default as PhNumberSquareThree } from './components/PhNumberSquareThree.vue.js';
export { default as PhNumberSquareTwo } from './components/PhNumberSquareTwo.vue.js';
export { default as PhNumberSquareZero } from './components/PhNumberSquareZero.vue.js';
export { default as PhNumberThree } from './components/PhNumberThree.vue.js';
export { default as PhNumberTwo } from './components/PhNumberTwo.vue.js';
export { default as PhNumberZero } from './components/PhNumberZero.vue.js';
export { default as PhNut } from './components/PhNut.vue.js';
export { default as PhNyTimesLogo } from './components/PhNyTimesLogo.vue.js';
export { default as PhOctagon } from './components/PhOctagon.vue.js';
export { default as PhOption } from './components/PhOption.vue.js';
export { default as PhPackage } from './components/PhPackage.vue.js';
export { default as PhPaintBrush } from './components/PhPaintBrush.vue.js';
export { default as PhPaintBrushBroad } from './components/PhPaintBrushBroad.vue.js';
export { default as PhPaintBrushHousehold } from './components/PhPaintBrushHousehold.vue.js';
export { default as PhPaintBucket } from './components/PhPaintBucket.vue.js';
export { default as PhPaintRoller } from './components/PhPaintRoller.vue.js';
export { default as PhPalette } from './components/PhPalette.vue.js';
export { default as PhPaperPlane } from './components/PhPaperPlane.vue.js';
export { default as PhPaperPlaneRight } from './components/PhPaperPlaneRight.vue.js';
export { default as PhPaperPlaneTilt } from './components/PhPaperPlaneTilt.vue.js';
export { default as PhPaperclip } from './components/PhPaperclip.vue.js';
export { default as PhPaperclipHorizontal } from './components/PhPaperclipHorizontal.vue.js';
export { default as PhParachute } from './components/PhParachute.vue.js';
export { default as PhPassword } from './components/PhPassword.vue.js';
export { default as PhPath } from './components/PhPath.vue.js';
export { default as PhPause } from './components/PhPause.vue.js';
export { default as PhPauseCircle } from './components/PhPauseCircle.vue.js';
export { default as PhPawPrint } from './components/PhPawPrint.vue.js';
export { default as PhPeace } from './components/PhPeace.vue.js';
export { default as PhPen } from './components/PhPen.vue.js';
export { default as PhPenNib } from './components/PhPenNib.vue.js';
export { default as PhPenNibStraight } from './components/PhPenNibStraight.vue.js';
export { default as PhPencil } from './components/PhPencil.vue.js';
export { default as PhPencilCircle } from './components/PhPencilCircle.vue.js';
export { default as PhPencilLine } from './components/PhPencilLine.vue.js';
export { default as PhPencilSimple } from './components/PhPencilSimple.vue.js';
export { default as PhPencilSimpleLine } from './components/PhPencilSimpleLine.vue.js';
export { default as PhPercent } from './components/PhPercent.vue.js';
export { default as PhPerson } from './components/PhPerson.vue.js';
export { default as PhPersonSimple } from './components/PhPersonSimple.vue.js';
export { default as PhPersonSimpleRun } from './components/PhPersonSimpleRun.vue.js';
export { default as PhPersonSimpleWalk } from './components/PhPersonSimpleWalk.vue.js';
export { default as PhPerspective } from './components/PhPerspective.vue.js';
export { default as PhPhone } from './components/PhPhone.vue.js';
export { default as PhPhoneCall } from './components/PhPhoneCall.vue.js';
export { default as PhPhoneDisconnect } from './components/PhPhoneDisconnect.vue.js';
export { default as PhPhoneIncoming } from './components/PhPhoneIncoming.vue.js';
export { default as PhPhoneOutgoing } from './components/PhPhoneOutgoing.vue.js';
export { default as PhPhoneSlash } from './components/PhPhoneSlash.vue.js';
export { default as PhPhoneX } from './components/PhPhoneX.vue.js';
export { default as PhPhosphorLogo } from './components/PhPhosphorLogo.vue.js';
export { default as PhPianoKeys } from './components/PhPianoKeys.vue.js';
export { default as PhPictureInPicture } from './components/PhPictureInPicture.vue.js';
export { default as PhPill } from './components/PhPill.vue.js';
export { default as PhPinterestLogo } from './components/PhPinterestLogo.vue.js';
export { default as PhPinwheel } from './components/PhPinwheel.vue.js';
export { default as PhPizza } from './components/PhPizza.vue.js';
export { default as PhPlaceholder } from './components/PhPlaceholder.vue.js';
export { default as PhPlanet } from './components/PhPlanet.vue.js';
export { default as PhPlay } from './components/PhPlay.vue.js';
export { default as PhPlayCircle } from './components/PhPlayCircle.vue.js';
export { default as PhPlaylist } from './components/PhPlaylist.vue.js';
export { default as PhPlug } from './components/PhPlug.vue.js';
export { default as PhPlugs } from './components/PhPlugs.vue.js';
export { default as PhPlugsConnected } from './components/PhPlugsConnected.vue.js';
export { default as PhPlus } from './components/PhPlus.vue.js';
export { default as PhPlusCircle } from './components/PhPlusCircle.vue.js';
export { default as PhPlusMinus } from './components/PhPlusMinus.vue.js';
export { default as PhPokerChip } from './components/PhPokerChip.vue.js';
export { default as PhPoliceCar } from './components/PhPoliceCar.vue.js';
export { default as PhPolygon } from './components/PhPolygon.vue.js';
export { default as PhPopcorn } from './components/PhPopcorn.vue.js';
export { default as PhPower } from './components/PhPower.vue.js';
export { default as PhPrescription } from './components/PhPrescription.vue.js';
export { default as PhPresentation } from './components/PhPresentation.vue.js';
export { default as PhPresentationChart } from './components/PhPresentationChart.vue.js';
export { default as PhPrinter } from './components/PhPrinter.vue.js';
export { default as PhProhibit } from './components/PhProhibit.vue.js';
export { default as PhProhibitInset } from './components/PhProhibitInset.vue.js';
export { default as PhProjectorScreen } from './components/PhProjectorScreen.vue.js';
export { default as PhProjectorScreenChart } from './components/PhProjectorScreenChart.vue.js';
export { default as PhPushPin } from './components/PhPushPin.vue.js';
export { default as PhPushPinSimple } from './components/PhPushPinSimple.vue.js';
export { default as PhPushPinSimpleSlash } from './components/PhPushPinSimpleSlash.vue.js';
export { default as PhPushPinSlash } from './components/PhPushPinSlash.vue.js';
export { default as PhPuzzlePiece } from './components/PhPuzzlePiece.vue.js';
export { default as PhQrCode } from './components/PhQrCode.vue.js';
export { default as PhQuestion } from './components/PhQuestion.vue.js';
export { default as PhQueue } from './components/PhQueue.vue.js';
export { default as PhQuotes } from './components/PhQuotes.vue.js';
export { default as PhRadical } from './components/PhRadical.vue.js';
export { default as PhRadio } from './components/PhRadio.vue.js';
export { default as PhRadioButton } from './components/PhRadioButton.vue.js';
export { default as PhRainbow } from './components/PhRainbow.vue.js';
export { default as PhRainbowCloud } from './components/PhRainbowCloud.vue.js';
export { default as PhReceipt } from './components/PhReceipt.vue.js';
export { default as PhRecord } from './components/PhRecord.vue.js';
export { default as PhRectangle } from './components/PhRectangle.vue.js';
export { default as PhRecycle } from './components/PhRecycle.vue.js';
export { default as PhRedditLogo } from './components/PhRedditLogo.vue.js';
export { default as PhRepeat } from './components/PhRepeat.vue.js';
export { default as PhRepeatOnce } from './components/PhRepeatOnce.vue.js';
export { default as PhRewind } from './components/PhRewind.vue.js';
export { default as PhRewindCircle } from './components/PhRewindCircle.vue.js';
export { default as PhRobot } from './components/PhRobot.vue.js';
export { default as PhRocket } from './components/PhRocket.vue.js';
export { default as PhRocketLaunch } from './components/PhRocketLaunch.vue.js';
export { default as PhRows } from './components/PhRows.vue.js';
export { default as PhRss } from './components/PhRss.vue.js';
export { default as PhRssSimple } from './components/PhRssSimple.vue.js';
export { default as PhRug } from './components/PhRug.vue.js';
export { default as PhRuler } from './components/PhRuler.vue.js';
export { default as PhScales } from './components/PhScales.vue.js';
export { default as PhScan } from './components/PhScan.vue.js';
export { default as PhScissors } from './components/PhScissors.vue.js';
export { default as PhScreencast } from './components/PhScreencast.vue.js';
export { default as PhScribbleLoop } from './components/PhScribbleLoop.vue.js';
export { default as PhScroll } from './components/PhScroll.vue.js';
export { default as PhSelection } from './components/PhSelection.vue.js';
export { default as PhSelectionAll } from './components/PhSelectionAll.vue.js';
export { default as PhSelectionBackground } from './components/PhSelectionBackground.vue.js';
export { default as PhSelectionForeground } from './components/PhSelectionForeground.vue.js';
export { default as PhSelectionInverse } from './components/PhSelectionInverse.vue.js';
export { default as PhSelectionPlus } from './components/PhSelectionPlus.vue.js';
export { default as PhSelectionSlash } from './components/PhSelectionSlash.vue.js';
export { default as PhShare } from './components/PhShare.vue.js';
export { default as PhShareNetwork } from './components/PhShareNetwork.vue.js';
export { default as PhShield } from './components/PhShield.vue.js';
export { default as PhShieldCheck } from './components/PhShieldCheck.vue.js';
export { default as PhShieldCheckered } from './components/PhShieldCheckered.vue.js';
export { default as PhShieldChevron } from './components/PhShieldChevron.vue.js';
export { default as PhShieldPlus } from './components/PhShieldPlus.vue.js';
export { default as PhShieldSlash } from './components/PhShieldSlash.vue.js';
export { default as PhShieldStar } from './components/PhShieldStar.vue.js';
export { default as PhShieldWarning } from './components/PhShieldWarning.vue.js';
export { default as PhShoppingBag } from './components/PhShoppingBag.vue.js';
export { default as PhShoppingBagOpen } from './components/PhShoppingBagOpen.vue.js';
export { default as PhShoppingCart } from './components/PhShoppingCart.vue.js';
export { default as PhShoppingCartSimple } from './components/PhShoppingCartSimple.vue.js';
export { default as PhShower } from './components/PhShower.vue.js';
export { default as PhShuffle } from './components/PhShuffle.vue.js';
export { default as PhShuffleAngular } from './components/PhShuffleAngular.vue.js';
export { default as PhShuffleSimple } from './components/PhShuffleSimple.vue.js';
export { default as PhSidebar } from './components/PhSidebar.vue.js';
export { default as PhSidebarSimple } from './components/PhSidebarSimple.vue.js';
export { default as PhSignIn } from './components/PhSignIn.vue.js';
export { default as PhSignOut } from './components/PhSignOut.vue.js';
export { default as PhSignpost } from './components/PhSignpost.vue.js';
export { default as PhSimCard } from './components/PhSimCard.vue.js';
export { default as PhSketchLogo } from './components/PhSketchLogo.vue.js';
export { default as PhSkipBack } from './components/PhSkipBack.vue.js';
export { default as PhSkipBackCircle } from './components/PhSkipBackCircle.vue.js';
export { default as PhSkipForward } from './components/PhSkipForward.vue.js';
export { default as PhSkipForwardCircle } from './components/PhSkipForwardCircle.vue.js';
export { default as PhSkull } from './components/PhSkull.vue.js';
export { default as PhSlackLogo } from './components/PhSlackLogo.vue.js';
export { default as PhSliders } from './components/PhSliders.vue.js';
export { default as PhSlidersHorizontal } from './components/PhSlidersHorizontal.vue.js';
export { default as PhSmiley } from './components/PhSmiley.vue.js';
export { default as PhSmileyBlank } from './components/PhSmileyBlank.vue.js';
export { default as PhSmileyMeh } from './components/PhSmileyMeh.vue.js';
export { default as PhSmileyNervous } from './components/PhSmileyNervous.vue.js';
export { default as PhSmileySad } from './components/PhSmileySad.vue.js';
export { default as PhSmileySticker } from './components/PhSmileySticker.vue.js';
export { default as PhSmileyWink } from './components/PhSmileyWink.vue.js';
export { default as PhSmileyXEyes } from './components/PhSmileyXEyes.vue.js';
export { default as PhSnapchatLogo } from './components/PhSnapchatLogo.vue.js';
export { default as PhSnowflake } from './components/PhSnowflake.vue.js';
export { default as PhSoccerBall } from './components/PhSoccerBall.vue.js';
export { default as PhSortAscending } from './components/PhSortAscending.vue.js';
export { default as PhSortDescending } from './components/PhSortDescending.vue.js';
export { default as PhSpade } from './components/PhSpade.vue.js';
export { default as PhSparkle } from './components/PhSparkle.vue.js';
export { default as PhSpeakerHigh } from './components/PhSpeakerHigh.vue.js';
export { default as PhSpeakerLow } from './components/PhSpeakerLow.vue.js';
export { default as PhSpeakerNone } from './components/PhSpeakerNone.vue.js';
export { default as PhSpeakerSimpleHigh } from './components/PhSpeakerSimpleHigh.vue.js';
export { default as PhSpeakerSimpleLow } from './components/PhSpeakerSimpleLow.vue.js';
export { default as PhSpeakerSimpleNone } from './components/PhSpeakerSimpleNone.vue.js';
export { default as PhSpeakerSimpleSlash } from './components/PhSpeakerSimpleSlash.vue.js';
export { default as PhSpeakerSimpleX } from './components/PhSpeakerSimpleX.vue.js';
export { default as PhSpeakerSlash } from './components/PhSpeakerSlash.vue.js';
export { default as PhSpeakerX } from './components/PhSpeakerX.vue.js';
export { default as PhSpinner } from './components/PhSpinner.vue.js';
export { default as PhSpinnerGap } from './components/PhSpinnerGap.vue.js';
export { default as PhSpiral } from './components/PhSpiral.vue.js';
export { default as PhSpotifyLogo } from './components/PhSpotifyLogo.vue.js';
export { default as PhSquare } from './components/PhSquare.vue.js';
export { default as PhSquareHalf } from './components/PhSquareHalf.vue.js';
export { default as PhSquareHalfBottom } from './components/PhSquareHalfBottom.vue.js';
export { default as PhSquareLogo } from './components/PhSquareLogo.vue.js';
export { default as PhSquaresFour } from './components/PhSquaresFour.vue.js';
export { default as PhStack } from './components/PhStack.vue.js';
export { default as PhStackOverflowLogo } from './components/PhStackOverflowLogo.vue.js';
export { default as PhStackSimple } from './components/PhStackSimple.vue.js';
export { default as PhStamp } from './components/PhStamp.vue.js';
export { default as PhStar } from './components/PhStar.vue.js';
export { default as PhStarFour } from './components/PhStarFour.vue.js';
export { default as PhStarHalf } from './components/PhStarHalf.vue.js';
export { default as PhSticker } from './components/PhSticker.vue.js';
export { default as PhStop } from './components/PhStop.vue.js';
export { default as PhStopCircle } from './components/PhStopCircle.vue.js';
export { default as PhStorefront } from './components/PhStorefront.vue.js';
export { default as PhStrategy } from './components/PhStrategy.vue.js';
export { default as PhStripeLogo } from './components/PhStripeLogo.vue.js';
export { default as PhStudent } from './components/PhStudent.vue.js';
export { default as PhSuitcase } from './components/PhSuitcase.vue.js';
export { default as PhSuitcaseSimple } from './components/PhSuitcaseSimple.vue.js';
export { default as PhSun } from './components/PhSun.vue.js';
export { default as PhSunDim } from './components/PhSunDim.vue.js';
export { default as PhSunHorizon } from './components/PhSunHorizon.vue.js';
export { default as PhSunglasses } from './components/PhSunglasses.vue.js';
export { default as PhSwap } from './components/PhSwap.vue.js';
export { default as PhSwatches } from './components/PhSwatches.vue.js';
export { default as PhSword } from './components/PhSword.vue.js';
export { default as PhSyringe } from './components/PhSyringe.vue.js';
export { default as PhTShirt } from './components/PhTShirt.vue.js';
export { default as PhTable } from './components/PhTable.vue.js';
export { default as PhTabs } from './components/PhTabs.vue.js';
export { default as PhTag } from './components/PhTag.vue.js';
export { default as PhTagChevron } from './components/PhTagChevron.vue.js';
export { default as PhTagSimple } from './components/PhTagSimple.vue.js';
export { default as PhTarget } from './components/PhTarget.vue.js';
export { default as PhTaxi } from './components/PhTaxi.vue.js';
export { default as PhTelegramLogo } from './components/PhTelegramLogo.vue.js';
export { default as PhTelevision } from './components/PhTelevision.vue.js';
export { default as PhTelevisionSimple } from './components/PhTelevisionSimple.vue.js';
export { default as PhTennisBall } from './components/PhTennisBall.vue.js';
export { default as PhTerminal } from './components/PhTerminal.vue.js';
export { default as PhTerminalWindow } from './components/PhTerminalWindow.vue.js';
export { default as PhTestTube } from './components/PhTestTube.vue.js';
export { default as PhTextAa } from './components/PhTextAa.vue.js';
export { default as PhTextAlignCenter } from './components/PhTextAlignCenter.vue.js';
export { default as PhTextAlignJustify } from './components/PhTextAlignJustify.vue.js';
export { default as PhTextAlignLeft } from './components/PhTextAlignLeft.vue.js';
export { default as PhTextAlignRight } from './components/PhTextAlignRight.vue.js';
export { default as PhTextBolder } from './components/PhTextBolder.vue.js';
export { default as PhTextH } from './components/PhTextH.vue.js';
export { default as PhTextHFive } from './components/PhTextHFive.vue.js';
export { default as PhTextHFour } from './components/PhTextHFour.vue.js';
export { default as PhTextHOne } from './components/PhTextHOne.vue.js';
export { default as PhTextHSix } from './components/PhTextHSix.vue.js';
export { default as PhTextHThree } from './components/PhTextHThree.vue.js';
export { default as PhTextHTwo } from './components/PhTextHTwo.vue.js';
export { default as PhTextIndent } from './components/PhTextIndent.vue.js';
export { default as PhTextItalic } from './components/PhTextItalic.vue.js';
export { default as PhTextOutdent } from './components/PhTextOutdent.vue.js';
export { default as PhTextStrikethrough } from './components/PhTextStrikethrough.vue.js';
export { default as PhTextT } from './components/PhTextT.vue.js';
export { default as PhTextUnderline } from './components/PhTextUnderline.vue.js';
export { default as PhTextbox } from './components/PhTextbox.vue.js';
export { default as PhThermometer } from './components/PhThermometer.vue.js';
export { default as PhThermometerCold } from './components/PhThermometerCold.vue.js';
export { default as PhThermometerHot } from './components/PhThermometerHot.vue.js';
export { default as PhThermometerSimple } from './components/PhThermometerSimple.vue.js';
export { default as PhThumbsDown } from './components/PhThumbsDown.vue.js';
export { default as PhThumbsUp } from './components/PhThumbsUp.vue.js';
export { default as PhTicket } from './components/PhTicket.vue.js';
export { default as PhTiktokLogo } from './components/PhTiktokLogo.vue.js';
export { default as PhTimer } from './components/PhTimer.vue.js';
export { default as PhToggleLeft } from './components/PhToggleLeft.vue.js';
export { default as PhToggleRight } from './components/PhToggleRight.vue.js';
export { default as PhToilet } from './components/PhToilet.vue.js';
export { default as PhToiletPaper } from './components/PhToiletPaper.vue.js';
export { default as PhTote } from './components/PhTote.vue.js';
export { default as PhToteSimple } from './components/PhToteSimple.vue.js';
export { default as PhTrademarkRegistered } from './components/PhTrademarkRegistered.vue.js';
export { default as PhTrafficCone } from './components/PhTrafficCone.vue.js';
export { default as PhTrafficSign } from './components/PhTrafficSign.vue.js';
export { default as PhTrafficSignal } from './components/PhTrafficSignal.vue.js';
export { default as PhTrain } from './components/PhTrain.vue.js';
export { default as PhTrainRegional } from './components/PhTrainRegional.vue.js';
export { default as PhTrainSimple } from './components/PhTrainSimple.vue.js';
export { default as PhTranslate } from './components/PhTranslate.vue.js';
export { default as PhTrash } from './components/PhTrash.vue.js';
export { default as PhTrashSimple } from './components/PhTrashSimple.vue.js';
export { default as PhTray } from './components/PhTray.vue.js';
export { default as PhTree } from './components/PhTree.vue.js';
export { default as PhTreeEvergreen } from './components/PhTreeEvergreen.vue.js';
export { default as PhTreeStructure } from './components/PhTreeStructure.vue.js';
export { default as PhTrendDown } from './components/PhTrendDown.vue.js';
export { default as PhTrendUp } from './components/PhTrendUp.vue.js';
export { default as PhTriangle } from './components/PhTriangle.vue.js';
export { default as PhTrophy } from './components/PhTrophy.vue.js';
export { default as PhTruck } from './components/PhTruck.vue.js';
export { default as PhTwitchLogo } from './components/PhTwitchLogo.vue.js';
export { default as PhTwitterLogo } from './components/PhTwitterLogo.vue.js';
export { default as PhUmbrella } from './components/PhUmbrella.vue.js';
export { default as PhUmbrellaSimple } from './components/PhUmbrellaSimple.vue.js';
export { default as PhUpload } from './components/PhUpload.vue.js';
export { default as PhUploadSimple } from './components/PhUploadSimple.vue.js';
export { default as PhUser } from './components/PhUser.vue.js';
export { default as PhUserCircle } from './components/PhUserCircle.vue.js';
export { default as PhUserCircleGear } from './components/PhUserCircleGear.vue.js';
export { default as PhUserCircleMinus } from './components/PhUserCircleMinus.vue.js';
export { default as PhUserCirclePlus } from './components/PhUserCirclePlus.vue.js';
export { default as PhUserFocus } from './components/PhUserFocus.vue.js';
export { default as PhUserGear } from './components/PhUserGear.vue.js';
export { default as PhUserList } from './components/PhUserList.vue.js';
export { default as PhUserMinus } from './components/PhUserMinus.vue.js';
export { default as PhUserPlus } from './components/PhUserPlus.vue.js';
export { default as PhUserRectangle } from './components/PhUserRectangle.vue.js';
export { default as PhUserSquare } from './components/PhUserSquare.vue.js';
export { default as PhUserSwitch } from './components/PhUserSwitch.vue.js';
export { default as PhUsers } from './components/PhUsers.vue.js';
export { default as PhUsersFour } from './components/PhUsersFour.vue.js';
export { default as PhUsersThree } from './components/PhUsersThree.vue.js';
export { default as PhVault } from './components/PhVault.vue.js';
export { default as PhVibrate } from './components/PhVibrate.vue.js';
export { default as PhVideoCamera } from './components/PhVideoCamera.vue.js';
export { default as PhVideoCameraSlash } from './components/PhVideoCameraSlash.vue.js';
export { default as PhVignette } from './components/PhVignette.vue.js';
export { default as PhVoicemail } from './components/PhVoicemail.vue.js';
export { default as PhVolleyball } from './components/PhVolleyball.vue.js';
export { default as PhWall } from './components/PhWall.vue.js';
export { default as PhWallet } from './components/PhWallet.vue.js';
export { default as PhWarning } from './components/PhWarning.vue.js';
export { default as PhWarningCircle } from './components/PhWarningCircle.vue.js';
export { default as PhWarningOctagon } from './components/PhWarningOctagon.vue.js';
export { default as PhWatch } from './components/PhWatch.vue.js';
export { default as PhWaveSawtooth } from './components/PhWaveSawtooth.vue.js';
export { default as PhWaveSine } from './components/PhWaveSine.vue.js';
export { default as PhWaveSquare } from './components/PhWaveSquare.vue.js';
export { default as PhWaveTriangle } from './components/PhWaveTriangle.vue.js';
export { default as PhWaves } from './components/PhWaves.vue.js';
export { default as PhWebcam } from './components/PhWebcam.vue.js';
export { default as PhWhatsappLogo } from './components/PhWhatsappLogo.vue.js';
export { default as PhWheelchair } from './components/PhWheelchair.vue.js';
export { default as PhWifiHigh } from './components/PhWifiHigh.vue.js';
export { default as PhWifiLow } from './components/PhWifiLow.vue.js';
export { default as PhWifiMedium } from './components/PhWifiMedium.vue.js';
export { default as PhWifiNone } from './components/PhWifiNone.vue.js';
export { default as PhWifiSlash } from './components/PhWifiSlash.vue.js';
export { default as PhWifiX } from './components/PhWifiX.vue.js';
export { default as PhWind } from './components/PhWind.vue.js';
export { default as PhWindowsLogo } from './components/PhWindowsLogo.vue.js';
export { default as PhWine } from './components/PhWine.vue.js';
export { default as PhWrench } from './components/PhWrench.vue.js';
export { default as PhX } from './components/PhX.vue.js';
export { default as PhXCircle } from './components/PhXCircle.vue.js';
export { default as PhXSquare } from './components/PhXSquare.vue.js';
export { default as PhYinYang } from './components/PhYinYang.vue.js';
export { default as PhYoutubeLogo } from './components/PhYoutubeLogo.vue.js';

// Import vue components

const install = function installPhosphorVue(Vue) {
  Object.entries(index).forEach(_ref => {
    let [componentName, component] = _ref;
    Vue.component(componentName, component);
  });
}; // Create module definition for Vue.use()

export { install as default };
